roles/openshift_version/tasks/main.yml

```diff
@@ -86,8 +86,14 @@
   include: set_version_rpm.yml
   when: not is_containerized | bool
 
-- name: Set openshift_version for containerized installation
-  include: set_version_containerized.yml
+- block:
+  - name: Set openshift_version for containerized installation
+    include: set_version_containerized.yml
+  - name: Determine openshift rpm version
+    include: rpm_version.yml
+  - name: Report rpm version and docker image version
+    debug:
+      msg: "openshift_rpm_version: {{ openshift_rpm_version }}, openshift_containerized_version: {{ openshift_version }}"
```

Comment:
The use of `openshift_containerized_version` makes it look like that should be a var name somewhere, when it is actually not.  Would it also make sense to fail/assert here if they are not the same?